src/serlo-editor/plugins/multimedia/toolbar/toolbar-wrapper.tsx

```diff
@@ -0,0 +1,51 @@
+import { faCog } from '@fortawesome/free-solid-svg-icons'
+import { ReactNode, useState } from 'react'
+
+import { FaIcon } from '@/components/fa-icon'
+import { ModalWithCloseButton } from '@/components/modal-with-close-button'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { PluginToolbar } from '@/serlo-editor/editor-ui/plugin-toolbar'
+import { PluginDefaultTools } from '@/serlo-editor/editor-ui/plugin-toolbar/plugin-tool-menu/plugin-default-tools'
+import { EditorPluginType } from '@/serlo-editor-integration/types/editor-plugin-type'
+
+interface Props {
```

Comment:
take it or leave it. We stick quite well to the pattern that the interface for `ToolbarWrapper` is called `ToolbarWrapperProps`